docs/theming/dark-mode.md

```diff
@@ -14,76 +17,166 @@ inlineHtmlPreviews: true
 :::
 
-import AutomaticDarkMode from '@site/static/usage/v8/theming/automatic-dark-mode/index.md';
+import SystemDarkMode from '@site/static/usage/v8/theming/system-dark-mode/index.md';
+
+<SystemDarkMode />
 
-<AutomaticDarkMode />
+### Class
 
-### Manually Toggle Dark Mode
+While the previous approaches are excellent for enabling the dark theme through file imports alone, there are scenarios where you may need more control over its application. In cases where you need to apply the dark theme conditionally, such as through a toggle, or if you want to extend the functionality based on system settings, we provide a dark theme class file. This file applies the dark theme when a specific class is added to an app. Importing the following stylesheet into the appropriate file will provide the necessary styles for using the dark theme with the class:
 
-In addition to adding the `dark` class to the `<body>` when the media query changes, the class can be added by the app, such as when a user changes a toggle, to switch between the light and dark themes:
+<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+
+<TabItem value="angular">
+
+```css
+@import '~@ionic/angular/css/themes/dark.class.css';
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```ts
+import '@ionic/core/css/themes/dark.class.css';
+```
+
+</TabItem>
+<TabItem value="react">
+
+```tsx
+import '@ionic/react/css/themes/dark.class.css';
+```
+
+</TabItem>
+<TabItem value="vue">
+
+```ts
+import '@ionic/vue/css/themes/dark.class.css';
+```
 
-import ManualDarkMode from '@site/static/usage/v8/theming/manual-dark-mode/index.md';
+</TabItem>
 
-<ManualDarkMode />
+</Tabs>
+
+This sets the [application colors](/docs/theming/themes#application-colors) and [stepped colors](/docs/theming/themes#stepped-colors) on the `.ion-theme-dark` selector, which must be applied to the app by the developer.
+
+:::caution Important
+The `.ion-theme-dark` class **must** be added to the `html` element in order to work with the imported dark theme.
+:::
+
+:::info Demo
+This demo combines site settings, system settings, and the toggle to decide when to show dark mode. The site's theme takes precedence over system settings. If your system settings differ from the site's theme when the demo loads, it will use the site's theme.
+
+Not sure how to change the system settings? Here's how to enable dark mode on [Windows 11](https://support.microsoft.com/en-us/windows/change-colors-in-windows-d26ef4d6-819a-581c-1581-493cfcc005fe) and on [macOS](https://support.apple.com/en-us/HT208976).
+:::
+
+import ClassDarkMode from '@site/static/usage/v8/theming/class-dark-mode/index.md';
+
+<ClassDarkMode />
```

Comment:
I'm a bit confused as why the demo is using  the system settings. As a user, I would expect that the demo would only have the toggle to display how adding/removing the class causing the app's theme to change.

Reply:
This was something [outlined in the design document](https://github.com/ionic-team/ionic-framework-design-documents/blob/main/projects/ionic-framework/theming/0002-distributing-theme-updates.md#documentation). I believe it was specifically requested to demonstrate to developers how they can expand upon reading from the system. Previously, we had these demos split up so that the first demo showed how to add the class based on the system settings, and then the second demo added the class based on the system settings along with a toggle. Since we offer the theme file for setting it based on the system, I used both the system and toggle for the class.